.build/colors.mk

```diff
@@ -1,18 +0,0 @@
-# Colorization rules
```

Comment:
I'm not saying I wasn't impressed :)

I'm really hoping we get to a place where we mostly just `go test ./...` and the Makefile's for new contributors and Travis.